adapters/element/index.js

```diff
@@ -0,0 +1,129 @@
+            },
+        );
+        const symbol = resp.data;
+        return symbol;
+    };
+    getPrice = async timestamp => {
+        const resp = await axios.get(
+            `${URL}/token-price?key=${KEY}&token_address=${this.token}&protocol=${this.protocol}&timestamp=${timestamp}`,
+            {
+                headers: {
+                    "User-Agent":
+                        "Mozilla/5.0 (Macintosh; Intel Mac OS X 11_4) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/93.0.4619.141 Safari/537.36",
+                },
+            },
+        );
+        return resp.data;
+    };
+    stop = async () => {
+        this.sdk.stop();
+    };
+
+    // getBuyer = async event => {
+    //     const buyer = event.returnValues.owner;
+    //     if (event.event === "NameRenewed") {
+    //         const txReceipt = await this.sdk.getTransactionReceipt(event.transactionHash);
+    //         if (txReceipt === null) {
+    //             return null;
+    //         }
+    //         return txReceipt.from;
+    //     }
+    //     return buyer;
+    // };
+
+    process = async event => {
+        const isER721 = event.event === "ERC721SellOrderFilled" || event.event === "ERC721BuyOrderFilled";
+        const isSellOrder = ["ERC721SellOrderFilled", "ERC1155SellOrderFilled"].includes(event.event);
+        const block = await this.sdk.getBlock(event.blockNumber);
+        const timestamp = moment.unix(block.timestamp).utc();
+        const po = await this.getPrice(block.timestamp);
+        const amount = isER721 ? 1 : event.returnValues["erc1155FillAmount"];
+        const price = isER721 ? event.returnValues["erc20TokenAmount"] : event.returnValues["erc20FillAmount"];
+        const nativePrice = new BigNumber(price).dividedBy(10 ** this.symbol.decimals);
+        const maker = event.returnValues["maker"];
+        const taker = event.returnValues["taker"];
+        const buyer = isSellOrder ? taker : maker;
+        const seller = isSellOrder ? maker : taker;
+        const nft_contract = event.returnValues["erc721Token"] || event.returnValues["erc1155Token"];
+        const tokenId = event.returnValues["erc721TokenId"] || event.returnValues["erc1155TokenId"];
+        const erc20token = event.returnValues["erc20Token"];
+        const token =
+            erc20token.toLowerCase() === "0xeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee"
+                ? "0x0000000000000000000000000000000000000000"
+                : erc20token;
+        const entity = {
+            provider_name: this.name, // the name of the folder
+            provider_contract: this.contract, // the providers contract from which you get data
+            protocol: this.protocol,
+            nft_contract, // : this.contract,
+            nft_id: tokenId,
+            token, // : this.token,
+            token_symbol: this.symbol.symbol,
+            amount,
+            price: nativePrice.toNumber(),
+            price_usd: nativePrice.multipliedBy(po.price).toNumber(),
```

Comment:
`token_symbol` is undefined
`price` and `price_usd` are `NaN`
Make sure that tests pass before requesting review

Reply:
I passed the test before submitting pr， script：
`node ./tester/test.js ../adapters/element/index.js`

and it returned the following result:
```
➜  nft-sales-adapters git:(main) ✗ node ./tester/test.js ../adapters/element/index.js
processing past blocks for element from block 15080677 to block 15080777
found 1 events for element in range from block 15080677 to 15080777
{
  provider_name: 'element',
  provider_contract: '0x20f780a973856b93f63670377900c1d2a50a77c4',
  protocol: 'ethereum',
  nft_contract: '0x52F687B1c6aACC92b47DA5209cf25D987C876628',
  nft_id: '756',
  token: '0x0000000000000000000000000000000000000000',
  token_symbol: 'eth',
  amount: 1,
  price: 0.01235,
  price_usd: 14.202253,
  seller: '0x9CdCbF212CCF4f11bcbC25CCde18fFCe886F0CA6',
  buyer: '0x633f6c7E25eE757d12643A32cE1586AC9e8542d5',
  sold_at: '2022-07-05 05:40:25',
  block_number: 15080677,
  transaction_hash: '0x3addc13601ada67456c10f5ab285714b74b8c16a43dcdc1fb45c3693b391a198'
}
processing past blocks for element from block 15080778 to block 15080877
processing past blocks for element from block 15080878 to block 15080977
processing past blocks for element from block 15080978 to block 15081077
processing past blocks for element from block 15081078 to block 15081177
processing past blocks for element from block 15081178 to block 15081277
found 1 events for element in range from block 15081178 to 15081277
{
  provider_name: 'element',
  provider_contract: '0x20f780a973856b93f63670377900c1d2a50a77c4',
  protocol: 'ethereum',
  nft_contract: '0x7Bd29408f11D2bFC23c34f18275bBf23bB716Bc7',
  nft_id: '11909',
  token: '0x0000000000000000000000000000000000000000',
  token_symbol: 'eth',
  amount: 1,
  price: 6.76,
  price_usd: 7872.4932,
  seller: '0xfb49BcFbAc5b105a78F29BC2a5E946681A088FB2',
  buyer: '0x070bF44247DDb3Ab3861E4E378631bcD68Ac3dd6',
  sold_at: '2022-07-05 07:58:30',
  block_number: 15081271,
  transaction_hash: '0x6d9d6c9696ddfe80a04fddc1e223b14ed1e32551f71fc9c9e8e96d1cd08bc0f9'
}
processing past blocks for element from block 15081278 to block 15081377
processing past blocks for element from block 15081378 to block 15081477
found 1 events for element in range from block 15081378 to 15081477
{
  provider_name: 'element',
  provider_contract: '0x20f780a973856b93f63670377900c1d2a50a77c4',
  protocol: 'ethereum',
  nft_contract: '0x7Bd29408f11D2bFC23c34f18275bBf23bB716Bc7',
  nft_id: '8117',
  token: '0x0000000000000000000000000000000000000000',
  token_symbol: 'eth',
  amount: 1,
  price: 6.14,
  price_usd: 7110.734,
  seller: '0xfb49BcFbAc5b105a78F29BC2a5E946681A088FB2',
  buyer: '0xA06D6E45Ab7B5af232f91b96FFd342E5d32Bcb32',
  sold_at: '2022-07-05 08:38:09',
  block_number: 15081451,
  transaction_hash: '0x2f6908aa7f68dd7353df0405163850a85818bbbc3f0b2c53b6988180f146de0c'
}
processing past blocks for element from block 15081478 to block 15081577
processing past blocks for element from block 15081578 to block 15081677
processing past blocks for element from block 15081678 to block 15081777
processing past blocks for element from block 15081778 to block 15081877
processing past blocks for element from block 15081878 to block 15081977
processing past blocks for element from block 15081978 to block 15082077
processing past blocks for element from block 15082078 to block 15082177
found 2 events for element in range from block 15082078 to 15082177
{
  provider_name: 'element',
  provider_contract: '0x20f780a973856b93f63670377900c1d2a50a77c4',
  protocol: 'ethereum',
  nft_contract: '0x7Bd29408f11D2bFC23c34f18275bBf23bB716Bc7',
  nft_id: '13843',
  token: '0x0000000000000000000000000000000000000000',
  token_symbol: 'eth',
  amount: 1,
  price: 6.69,
  price_usd: 7555.4184,
  seller: '0xfb49BcFbAc5b105a78F29BC2a5E946681A088FB2',
  buyer: '0xC93c81E70A1e8e342206f4278e2fb6D2630f5E04',
  sold_at: '2022-07-05 11:04:44',
  block_number: 15082115,
  transaction_hash: '0x8d1997a5ac87e22853930517a91233393a80bedad7deef0ab6dd32bd66620b39'
}
{
  provider_name: 'element',
  provider_contract: '0x20f780a973856b93f63670377900c1d2a50a77c4',
  protocol: 'ethereum',
  nft_contract: '0x7Bd29408f11D2bFC23c34f18275bBf23bB716Bc7',
  nft_id: '8073',
  token: '0x0000000000000000000000000000000000000000',
  token_symbol: 'eth',
  amount: 1,
  price: 6.18,
  price_usd: 6979.4448,
  seller: '0xfb49BcFbAc5b105a78F29BC2a5E946681A088FB2',
  buyer: '0x567343b9e326F960e5F0fAEa33D79820B9AC73B8',
  sold_at: '2022-07-05 11:14:14',
  block_number: 15082157,
  transaction_hash: '0x1b56c4953ec8c79ff9b3cdfe0a291875ae13f75ef0affcdf575a83b530dc25ff'
}
```